superset/config.py

```diff
@@ -349,7 +349,7 @@ def _try_json_readsha(  # pylint: disable=unused-argument
     # by that custom datasource access. So we are assuming a default security config,
     # a custom security config could potentially give access to setting filters on
     # tables that users do not have access to.
-    "ROW_LEVEL_SECURITY": False,
+    "ROW_LEVEL_SECURITY": True,
```

Comment:
Maybe we could just remove this flag from the config, one less config key, and a bunch of `if`'s on the code

Reply:
I'll leave it for a short period just to make it easy for orgs running master to disable the feature in case we run into a nasty bug.